dpctl/tensor/_copy_utils.py

```diff
@@ -93,12 +93,12 @@ def copy_to_numpy(ary):
     )
 
 
-def copy_from_numpy(np_ary, usm_type="device", queue=None):
+def copy_from_numpy(np_ary, usm_type="device", sycl_queue=None):
```

Comment:
`sycl_queue`? Are you sure we will not be changing it again in couple of months 😄

Reply:
Maybe, but it has to stay consistent across API surface (we have `device` and `sycl_queue` in `asarray`, but `queue` in `from_numpy`).